src/features/game/events/landExpansion/utils.ts

```diff
@@ -11,5 +11,5 @@ export const getSupportedChickens = (state: Readonly<GameState>) => {
       (coop) => coop.readyAt < Date.now()
     ).length ?? 0;
 
-  return chickenCoop ? chickenHouses * 10 * 1.5 : chickenHouses * 10;
+  return chickenCoop ? henHouses * 15 : henHouses * 10;
```

Comment:
Do you think we could put the 1.5 or 15 into a constant so its not just a "magical" number. The constant can be `CHICKEN_COOP_MULTIPLIER` or something.